### DISCUSSION TITLE
[应用商店] 根据不同系统提供不同的软件版本

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [1Panel-dev/1Panel#5141] [Ideas]
### 1Panel 版本

1.10.9-TLS

### 请描述您的需求或者改进建议

根据不同的系统提供不同的软件版本。
例如OpenResty:
OpenResty为不同版本的Ubuntu/Debian提供了不同的版本(例如Ubuntu 22.04的版本号为Jammy,而20.04为Focal)。

### 请描述你建议的实现方案

允许用户在应用商店中选择不同版本(新下拉菜单选择),或者通过命令/读取文件查看系统版本从而由1Panel决定选择合适的版本。

### 附加信息

_这个issue大可以不作为开发重点（（（_
因为我的服务器是Ubuntu 22.04,而1Panel应用商店提供的版本是Focal(适用于20.04)。
目前还没出现因为未安装适用于自己系统的OpenResty而导致的严重问题。

**Comment by wanghe-fit2cloud** [comment]
宿主机操作系统和应用基础镜像操作系统没有直接关系，如果是单纯运行容器，可以忽略。
